src/Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue/Functions/OrderProcessingFunction.cs

```diff
@@ -13,14 +14,18 @@ namespace Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue.Function
     public class OrderProcessingFunction
     {
         private readonly IAzureServiceBusMessageRouter _messageRouter;
+        private readonly TelemetryClient _telemetryClient;
         private readonly string _jobId;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="OrderProcessingFunction" /> class.
         /// </summary>
-        public OrderProcessingFunction(IAzureServiceBusMessageRouter messageRouter)
+        public OrderProcessingFunction(
+            IAzureServiceBusMessageRouter messageRouter,
+            TelemetryClient telemetryClient)
         {
             _messageRouter = messageRouter;
+            _telemetryClient = telemetryClient;
```

Comment:
I think we should try to avoid to have telemetryClient instances here ?

Reply:
See comment here: https://github.com/arcus-azure/arcus.messaging/pull/344#discussion_r1000168182
Will be extracted from the function but was too much for this PR to also include this change. Not that this adaption is only for in-process Azure Functions. We will probably have to come up with something similar as with the HTTP trigger where we have an `HttpBasedFunction` base class that hides these things, or an additional type that hides the client.